src/domain/synthetics/markets/useMarketTokensAPR.ts

```diff
@@ -186,3 +171,20 @@ export function useMarketTokensAPR(chainId: number): MarketTokensAPRResult {
     avgMarketsAPR: data?.avgMarketsAPR,
   };
 }
+
+function calcAprByBorrowingFee(marketInfo: MarketInfo, poolValue: BigNumber) {
+  const longOi = marketInfo.longInterestUsd;
+  const shortOi = marketInfo.shortInterestUsd;
+  const isLongPayingBorrowingFee = longOi.gt(shortOi) ? true : false;
```

Comment:
i guess `longOi.gt(shortOi) ? true : false;` is the same as `longOi.gt(shortOi)`

Reply:
i guess longOi.gt(shortOi) ? true : false; is the same as longOi.gt(shortOi)